test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```

Comment:
Kindly include this file ` test.md ` in your `.gitignore` file. This file should not exist in the repo and should be ignored by git as it's in the requirement for this project.